lib/bas/domain/exceptions/function_not_implement.rb

```diff
@@ -7,7 +7,7 @@ module Exceptions
     # It inherits from StandardError # and allows developers to raise a specific exception when a required function
     # remains unimplemented in a subclass.
     #
-    class FunctionNotImplemented < StandardError
+    class FunctionNotImplement < StandardError
```

Comment:
Hello @L-Zuluaga. In the new version, we added a new error type called FunctionNotImplemented to handle errors in base files when a function is not defined. However,  a conflict occurred during testing because both the original file `lib/bas/domain/exceptions/function_not_implemented.rb` and the new file `lib/v2/utils/exceptions/function_not_implemented.rb` shared the same name. I thought, since the old version will be removed, to avoid the conflict, keep the coverage at 100%, and keep the name for the error for the new version, we can rename the old version from `FunctionNotImplemented` to `FunctionNotImplement`.

Reply:
Makes total sense! thanks for the clarification!